docs/source/plugin/bot.rst

```diff
@@ -96,6 +96,122 @@ the :meth:`~sopel.bot.SopelWrapper.action` method::
     bot.action('does something')
 
 
+Do it with style
+================
+
+.. Custom role definition to apply custom style to inline text
+
+.. role:: red
+    :class: red
+
+.. role:: strongred
+    :class: strong red
+
+.. role:: strike
+    :class: strike
+
+.. role:: strongem
+    :class: strong em
+
+.. role:: spoiler
+    :class: spoiler
+
+
+When the bot talks, replies, or acts, it can do it with style: colors,
+**bold**, *italic*, _underline_, :strike:`striked`, or ``monospace``. IRC
+formatting works with control code, bytes you can use to tell IRC clients how
+to display some part of the text.
+
+.. seealso::
+
+    If you want to know more about `IRC formatting in general`__ and some of
+    its limitations, the modern IRC documentation may be of interest to you.
+
+    .. __: https://modern.ircdocs.horse/formatting.html
```

Comment:
This just makes more sense as a link location. Alternative suggestion was to link the whole phrase, "more about IRC formatting in general".

```suggestion
    If you want to know more about IRC formatting in general and some of its
    limitations, `the modern IRC documentation`__ may be of interest to you.

    .. __: https://modern.ircdocs.horse/formatting.html
```